src/main/msp/msp.c

```diff
@@ -1371,6 +1371,7 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, (uint16_t)constrain(gpsSol.llh.altCm / 100, 0, UINT16_MAX)); // alt changed from 1m to 0.01m per lsb since MSP API 1.39 by RTH. To maintain backwards compatibility compensate to 1m per lsb in MSP again.
         sbufWriteU16(dst, gpsSol.groundSpeed);
         sbufWriteU16(dst, gpsSol.groundCourse);
+        sbufWriteU16(dst, gpsSol.hdop);    // Added in API version 1.44
```

Comment:
Sorry for bother with this again, but as I show you in the example I attached to my first review, the comment is OVER the new element added. 

This comment marks the start point of the 1.44 version, not only the line that you added. If someone adds more elements after yours before next release they will belong to 1.44 too.